src/sensr_client.cpp

```diff
@@ -26,45 +26,43 @@ namespace sensr
   }
 
   void Client::Reconnect() {
```

Comment:
Who is calling `Reconnect`? Is it called multiple times or just once?

Reply:
it is an interface for user. So the library user needs to call this. The intention seems that user call once. But they could run multiple times so that is way `is_reconnecting_` check condition block.